integ-test/src/test/java/org/opensearch/sql/legacy/AggregationExpressionIT.java

```diff
@@ -213,7 +213,7 @@ public void groupByDateShouldPass() {
         Index.BANK.getName()));
 
     verifySchema(response,
-        schema("birthdate", null, "timestamp"),
+        schema("birthdate", null, "date"),
```

Comment:
a breaking change?

Reply:
I may need help with this. I have tried to modify the implementation a few different ways, but no matter what I try I have failing tests about reported types somewhere else in the code.